index.html

```diff
@@ -3,9 +3,61 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style/index.css">
     <title>Document</title>
 </head>
 <body>
-    
+    <H1>Casey's Shop</H1>
+    <Section class='top-nav'>
+        <h3>Casey Boyland</h3>
+            <nav>
```

Comment:
awesome, you have a nav bar with "About", "Projects" and "Contact" - sweet sweet!